Filter icons for render mode (#120)
RedMadRobot/figma-export: CONFIG.md

    swiftUIImageSwift: "./Source/Image+extension_icons.swift"
    # [optional] Absolute or relative path to swift file where to generate extension for UIImage for accessing icons from the code (e.g. UIImage.ic24ArrowRight)
    imageSwift: "./Example/Source/UIImage+extension_icons.swift"
    # Asset render mode: "template", "original" or "default". Default value is "template".
    renderMode: default
    # Configure the suffix for filtering Icons and to denote a asset render mode: "default". 

>     # ❗️ It will work, if renderMode have default value is "template". Defaults to nil.

Reviewer: Предлагаю убрать эмоджи. Они отвлекают внимание на себя, хотя эта опция очень редко будет использоваться.

    renderModeDefaultSuffix: '_default'
    # Configure the suffix for filtering Icons and to denote a asset render mode: "original". 

>     # ❗️ It will work, if renderMode have default value is "template". Defaults to nil.

Reviewer: ```suggestion
    # It will work when renderMode value is "template". Defaults to nil.
```
Кажется вот так будет правильней с т.з. англ. яз.

    renderModeOriginalSuffix: '_original'
    # Configure the suffix for filtering Icons and to denote a asset render mode: "template". 
    # ❗️ It will work, if renderMode have default value isn't "template". Defaults to nil.

>     renderModeOriginalSuffix: '_template'

Reviewer: Тут опечатка. Должно быть `renderModeTemplateSuffix`.

Reviewer: Может быть убрать этот параметр? Свойство `renderMode` обозначает какой режим рендеринга использовать для всех иконок. То есть по умолчанию все иконки будут иметь template режим. Если кто-то хочет у нескольких иконок задать отключить template режим то он задаст свойство `renderModeOriginalSuffix`. Кажется что и `renderModeDefaultSuffix` не нужен.

Author: > Может быть убрать этот параметр? Свойство `renderMode` обозначает какой режим рендеринга использовать для всех иконок. То есть по умолчанию все иконки будут иметь template режим. Если кто-то хочет у нескольких иконок отключить template режим то он задаст свойство `renderModeOriginalSuffix`. Кажется что и `renderModeDefaultSuffix` не нужен.

Если цветные иконки расположены в одном файле, они уже имеют свой суффикс (для темной темы) - их будет невозможно отфильтровать, получается нужно задавать через свойство `renderMode` для всех  и фильтровать через `renderModeTemplateSuffix` оставшиеся. 
Возможны еще какие-то кейсы и кажется лучше больше параметров для большей гибкости, чем их отсутсвие.

Reviewer: То есть у и иконки темной темы будет двойной суффикс: `_dark_template`?

Author: Эм, нет. 
renderMode = default
Для темной темы будут иконки суффиксом: `_dark`
ЧБ иконки будут с суффиксом `_template` и их будем фильтровать через параметр `renderModeTemplateSuffix`

Reviewer: Да, сорри, не заметил отрицание isn't
